terminator/tests/path_test.rs

```diff
@@ -0,0 +1,38 @@
+use std::time::Duration;
+use tracing::{info, Level};
+use terminator::{platforms, AutomationError, Selector};
+
+#[tokio::test]
+async fn test_path() -> Result<(), AutomationError> {
+    tracing_subscriber::fmt::Subscriber::builder()
+        .with_max_level(Level::DEBUG)
+        .init();
+
+    const DEFAULT_FIND_TIMEOUT: Duration = Duration::from_millis(10000);
+    let timeout_ms = DEFAULT_FIND_TIMEOUT;
+
+    // replace with updated selector from your own ui tree
+    let sel = Selector::Path("/Window[3]/Custom[8]/Pane/Document/Group[2]/Group[1]/Group[3]/Document[1]/Button[5]".to_string());
```

Comment:
is that simialr synta to playwrhgit ?

https://playwright.dev/docs/locators#locate-by-css-or-xpath

ideally we mimic their synta